### PR TITLE
New Libre Office Writer Cheat Sheet

### PR DESCRIPTION
**Pull request by DPAUTH**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Libre Office Writer Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Provide an overview of the changes this pull request introduces.

###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/libre_office_writer_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@DPAUTH Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Libreoffice writer cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/libre_office_writer_cheat_sheet_mobile.png?MovJjaXsz9)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Libreoffice writer cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/libre_office_writer_cheat_sheet_mobile.png?Gfrckaeu1p)

**Comment by edgesince84** [comment]
thanks for the PR @DPAUTH - we will review shortly

**Comment by edgesince84** [comment]
hey @DPAUTH - I think this could look a little better if the "Shortcut Keys for LibreOffice Writer" section could be broken down into two parts, since it's a whole lot longer in the expanded view.

**Comment by DPAUTH** [comment]
@edgesince84  What name should I give to the two parts?

File: share/goodie/cheat_sheets/json/libre-office-writer.json [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+		"sourceName": "LibreOffice Help",
+		"sourceUrl": "https://help.libreoffice.org/Writer/Shortcut_Keys_for_Writer"
+	},
+	"aliases": [
+		"libreoffice writer",
+		"libre writer"
+	],
+	"template_type": "keyboard",
+	"section_order": [
+		"Function Keys for LibreOffice Writer",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/libre-office-writer.json]: "Function Keys" would be enough here

File: share/goodie/cheat_sheets/json/libre-office-writer.json [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+		"Shortcut Keys for LibreOffice Writer",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/libre-office-writer.json]: "Shortcut Keys" would be enough here

File: share/goodie/cheat_sheets/json/libre-office-writer.json [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+		"Shortcut Keys for Paragraphs and Heading Levels",
+		"Shortcut Keys for Tables in LibreOffice Writer",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/libre-office-writer.json]: "Shortcut Keys for Tables"

**Comment by DPAUTH** [comment]
@jophab  Thanks for the feedback, but I wanted the section names to be the same with the ones my source has. https://help.libreoffice.org/Writer/Shortcut_Keys_for_Writer  If you believe I should change the section names, I dont mind doing it. I still need @edgesince84 to tell me what name should I give to the 2 new sections that will be created when I break in 2 parts the  "Shortcut Keys for LibreOffice Writer" section

**Comment by DPAUTH** [comment]
@jophab  Thanks for the feedback, but I wanted the section names to be the same with the ones my source has. https://help.libreoffice.org/Writer/Shortcut_Keys_for_Writer  If you believe I should change the section names, I dont mind doing it. I still need @edgesince84 to tell me what name should I give to the 2 new sections that will be created when I break in 2 parts the  "Shortcut Keys for LibreOffice Writer" section

**Comment by DPAUTH** [comment]
Oops I accidentally closed the pull request.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Libreoffice writer cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/libre_office_writer_cheat_sheet_mobile.png?VuUh1qDorU)

**Pull request merged**